backend/handlers/institution_handler.py

```diff
@@ -74,6 +74,63 @@ def adminToJson(admin):
     }
     return Utils.toJson(admin_json)
 
+def config_user(user, data, institution):
```

Comment:
Esse método, tendo em vista que só atualiza o usuário, não faria mais sentido estar no modelo  de usuário?

Reply:
Maybe, but this method is an aux method to the handler with a very specific approach. It won't be used in anywhere else, only in institution_handler...